public void testIssue108() throws IOException {
    TestCase.test();
}
